assembl/static2/js/app/components/debate/survey/post.jsx

```diff
@@ -376,6 +416,11 @@ class Post extends React.Component<Props> {
           </div>
           {screenWidth < SMALL_SCREEN_WIDTH && (
             <div className="actions">
+              {!isPending ? (
```

Comment:
for the moment, i don't think we should: this would not be consistent with the other developements
by the way, i think that a const shareButton should be static, and should not contain logic, and its type should not be "or null"
imho, this makes sense to have if / else logic about wether or not displaying content in the template part of the render